### DISCUSSION TITLE
View different emoji each day based on mood

### DISCUSSION THREAD

**Discussion by mmfs37** [Richardsl/heatmap-calendar-obsidian#3] [Q&A] [answered]
I wish I could do a summarise your day dataview. I would pick each day a emoji that summarises the day. And in the heatmap for everyday it shows exactly that emoji. 
I know it is somehow possible. It would be at `content: ` but I don't know what to put in there. If I put data, p.Emo etc it doesnt work... What is the valid expression for my problem?

**Comment by Richardsl** [comment]
hi, sry but i'm not sure if i understand what you mean

**Reply by mmfs37** [reply]
for the heatmap I do something like [meditate=True] and then the heatmap shows me a green box with a checkmark in it. I can do that and it works. I want to do now [dailysymbol=🔅] and then the heatmap shows for the day a green box with that "🔅". How can I do this?

**Comment by Richardsl** [comment, marked as answer]
I don't have the ability to test the code where i am now, but maybe something like this should work:

```javascript

dv.span("**Meditate**")

const calendarData = { 
    entries: [] 
}

for(let page of dv.pages('"Journal"').where(p=>p.meditate || p.dailysymbol).sort(p=>p.file.name)){ 
    calendarData.entries.push({
        date: page.file.name, 
        intensity: page.meditate,
        content: page.dailysymbol,
    })
}

renderHeatmapCalendar(this.container, calendarData)

```

**Reply by mmfs37** [reply]
thought that I tried it that way but it works. Thanks a lot mate
